apps/src/blockly/cdoBlocklyWrapper.js

```diff
@@ -67,6 +68,25 @@ function strip(code) {
   );
 }
 
+/**
+ * Given a type string for a field input, returns an appropriate change handler function
+ * for that type, which customizes the input field and provides validation on blur.
+ * @param {string} type
+ * @returns {?function}
+ */
+function getFieldInputChangeHandler(type) {
+  const clampedNumberMatch = type.match(CLAMPED_NUMBER_REGEX);
+  if (clampedNumberMatch) {
+    const min = parseFloat(clampedNumberMatch[1]);
+    const max = parseFloat(clampedNumberMatch[2]);
+    return Blockly.FieldTextInput.clampedNumberValidator(min, max);
+  } else if ('Number' === type) {
```

Comment:
Ditto here: Could we use `Blockly.BlockValueType.NUMBER` instead of `'Number'`?

Non-blocking Nit: Would this be easier to read with the operators reversed (like it is in cdoUtils)?